packages/typescript-vfs/src/index.ts

```diff
@@ -555,14 +555,14 @@ export function createVirtualCompilerHost(sys: System, compilerOptions: Compiler
       // getDefaultLibLocation: () => '/',
       getDirectories: () => [],
       getNewLine: () => sys.newLine,
-      getSourceFile: fileName => {
+      getSourceFile: (fileName, languageVersionOrOptions) => {
         return (
           sourceFiles.get(fileName) ||
           save(
             ts.createSourceFile(
               fileName,
               sys.readFile(fileName)!,
-              compilerOptions.target || defaultCompilerOptions(ts).target!,
+              languageVersionOrOptions,
```

Comment:
I suspect this should be this for backwards compat:

```suggestion
              languageVersionOrOptions ?? compilerOptions.target ?? defaultCompilerOptions(ts).target!,
```

Reply:
(The website does old versions of TS where this may matter)